src/barplot.ts

```diff
@@ -345,7 +349,7 @@ export class Barplot {
 
   onMouseover(data: DataPoint, index: number) {
     assertType(this, Barplot);
-    const initialText = data.name + ": " + data.description;
+    const initialText = data.name === data.description ? data.name :  data.name + ": " + data.description;
```

Comment:
That's awesome, I love that your implementation still ensures that the first `Barplot()` retains it's original functionality while supporting some finer tuning on the second `Barplot()`. The ternary operator is a nice touch as well!